.gitignore

```diff
@@ -1,5 +1,5 @@
-*swp
-node_modules
-*.log
-.DS_Store
-.idea
+*swp
+node_modules
+*.log
+.DS_Store
+.idea
```

Comment:
can you keep the new line at the end of the file? Github seems to like it and puts a 🚫 if you don't, so I've adopted liking it too ^_^

I see it in a few other files, so just scroll this PR and you'll see where!